java/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -79,22 +97,3 @@ class ExternalApi extends Callable {
   /** Holds if this API is supported by existing CodeQL libraries, that is, it is either a recognized source or sink or has a flow summary. */
   predicate isSupported() { this.hasSummary() or this.isSource() or this.isSink() }
 }
-
-/** DEPRECATED: Alias for ExternalApi */
-deprecated class ExternalAPI = ExternalApi;
```

Comment:
This should not be removed.

Reply:
Alright I will put it back. Took the chance of removing it as it is a very recent deprecation (Erik Krogh made lots of refactoring to rename classes to use PascalCasing for classes).

Reply:
It is exactly there because, in principle, someone could be referring to the old name. So instead of breaking their code, they will get a warning.

Reply:
Oh right. Don't know what I was thinking.